.fixtures.yml

```diff
@@ -4,6 +4,6 @@ fixtures:
     java: 'https://github.com/puppetlabs/puppetlabs-java.git'
     stdlib: 'https://github.com/puppetlabs/puppetlabs-stdlib.git'
     zookeeper: 'https://github.com/deric/puppet-zookeeper.git'
-    systemd: 'https://github.com/camptocamp/puppet-systemd.git'
+    systemd: 'https://github.com/voxpupuli/puppet-systemd.git'
```

Comment:
This is already in the `master` branch :-)

```
git fetch --all
git rebase origin/master
git push -f
```

and we should be good!

Reply:
I  did it but there is nothing to update
```
$ git fetch --all
Fetching origin
$ git rebase origin/master
Current branch log4j is up to date.
$ git push -f
Everything up-to-date
```

Reply:
Maybe  it needs to be reverted?

Reply:
I forgot to fetch upstream :-) 
It should be ok

Reply:
No worries, thanks!

Reply:
thank you!